stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/attachments/components/MessageAttachmensContent.kt

```diff
@@ -0,0 +1,45 @@
+package io.getstream.chat.android.compose.ui.attachments.components
+
+import androidx.compose.foundation.layout.padding
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.unit.dp
+import io.getstream.chat.android.client.models.Message
+import io.getstream.chat.android.compose.state.messages.attachments.AttachmentState
+import io.getstream.chat.android.compose.state.messages.items.MessageItem
+import io.getstream.chat.android.compose.ui.theme.ChatTheme
+
+/**
+ * Represents the content that's shown in message attachments. We decide if we need to show link previews or other
+ * attachments.
+ *
+ * @param messageItem - The message that contains the attachments.
+ * @param onLongItemClick - Handler for long item taps on this content.
+ * */
+@Composable
+public fun MessageAttachmentsContent(
```

Comment:
Giphy attachments have something weird going on after the changes on this PR.

|develop|this branch|
|---|---|
| ![image](https://user-images.githubusercontent.com/12054216/130484297-d8c75fc5-54bc-4bc0-99e9-290feb9fa498.png)| ![image](https://user-images.githubusercontent.com/12054216/130484220-ac794af1-71d5-4c46-83da-86aef86b2148.png)|

Reply:
This is probably the same problem we had in core SDK - Giphy attachment has `titleLink != null` and we were rendering wrong view type